docs/tutorial.rst

```diff
@@ -324,10 +324,37 @@ Public Docker container on Github Packages
 mlf-core by default pushes the Docker container using the ``publish_docker.yml`` Github Actions workflow to `Github Packages <https://github.com/features/packages>`_.
 If you want to push your Docker container to a different registry you need to adapt the workflow and potentially update the username and add a Github secret for your password.
 By default, containers pushed to Github are private. As a result you would need to log in to pull the container.
+
+
 Hence, you have to make your Docker container public by navigating to the used Github account, selecting ``Packages`` and then your package.
-On the right you will find a button ``package settings``. Scroll down on the package settings page and at the bottom you will find a button ``make public``.
-Click it, authenticate and your Github container is now public!
-Be aware of the fact that building the Docker container usually takes 15-20 minutes and therefore your Docker container will not immidiately show up in the Packages tab.
+
+.. figure:: images/public_docker/step_1.png
+   :alt: Creating a public docker image
+
+As of writing this, there is a bug with the GitHub UI, that doesn't show private images without selecting the visibility. Click visibility, and then private, and select your docker image.
+
+.. figure:: images/public_docker/step_2.png
+   :alt: Private image bug
+
+.. figure:: images/public_docker/step_3.png
```

Comment:
It would be awesome if the bottom of this screenshot could be cropped a little bit. It has a lot of dead space.